tests/test-document-fuse.sh

```diff
@@ -49,7 +49,7 @@ export XDG_RUNTIME_DIR=${TEST_DATA_DIR}/runtime
 cleanup () {
     fusermount -u $XDG_RUNTIME_DIR/doc || :
     sleep 0.1
-    /bin/kill -9 $DBUS_SESSION_BUS_PID
+    kill -9 $DBUS_SESSION_BUS_PID
```

Comment:
This change seems fine. There are some corner-case differences between coreutils kill and bash's kill builtin, but they don't seem like they would be applicable here.

As a side suggestion, killing the dbus-daemon with the default `SIGTERM` will let it shut down gracefully (which matters in some rare situations involving non-abstract AF_UNIX sockets), and the variable is underquoted:

```suggestion
    kill "$DBUS_SESSION_BUS_PID"
```